Libraries/acdriver/CMakeLists.txt

```diff
@@ -23,7 +23,8 @@ find_package(PNG REQUIRED)
 
 target_link_libraries(acdriver PUBLIC xcassets util plist dependency ext bom car ${PNG_LIBRARIES})
 target_include_directories(acdriver PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/Headers")
-target_include_directories(acdriver PRIVATE "${PNG_INCLUDE_DIRS}")
+target_include_directories(acdriver PRIVATE ${PNG_INCLUDE_DIRS})
+add_definitions(${PNG_DEFINITIONS})
```

Comment:
nit: the more fine-grained way to do this is `set_target_properties()` with `COMPILE_DEFINITIONS`